tests/views/web_test.py

```diff
@@ -269,6 +274,61 @@ def test_index_with_recipient_and_message(self):
         self.assertHasCsrf(response.forms['send-love-form'], response.session)
 
 
+class SentTest(LoggedInUserBaseTest):
+    """
+    Testing the sent page
+    """
+
+    def setUp(self):
+        super(SentTest, self).setUp()
+        self.recipient = create_employee(username='janedoe')
+
+    def tearDown(self):
+        self.recipient.key.delete()
+        super(SentTest, self).tearDown()
+
+    def test_missing_args_is_redirect(self):
+        response = self.app.get('/sent')
+
+        self.assertEqual(response.status_int, 302)
+
+    def test_sent_with_args(self):
+        response = self.app.get('/sent', dict(recipients='janedoe', message='hi', link_id='cn23sx'))
+        self.assertIsNotNone(response.context['current_time'])
+        self.assertEqual(response.context['current_user'], self.current_user)
+        self.assertIsNotNone(response.context['loved'])
+        self.assertEqual(response.context['url'], config.APP_BASE_URL + 'l/cn23sx')
```

Comment:
This test is failing on TravisCI

`Traceback (most recent call last):
  File "/home/travis/build/Yelp/love/tests/views/web_test.py", line 300, in test_sent_with_args
    self.assertEqual(response.context['url'], config.APP_BASE_URL + 'l/cn23sx')
AssertionError: 'https://PROJECT_ID.appspot.com//l/cn23sx' != 'https://PROJECT_ID.appspot.com/l/cn23sx'`